resolwe_bio/descriptors/sample_template.yml

```diff
@@ -0,0 +1,348 @@
+- name: sample_annotation
+  label: GEO sample annotation
+  description: GEO sample annotation template
+  var:
+    - name: geo_annotation
+      label: GEO Annotation
+      group:
+        - name: description
+          label: Description
+          type: basic:string
+          required: false
+          description: >
+            Additional information not provided in the other fields, or paste in broad descriptions that cannot be easily dissected into the other fields.
+        - name: annotator
+          label: Annotator
+          type: basic:string
+        - name: organism
+          label: Organism
+          type: basic:string
+          description: >
+            Identify the organism from which the sequences were derived.
+          choices:
+            - label: Homo sapiens
+              value: Homo sapiens
+            - label: Mus musculus
+              value: Mus musculus
+            - label: Dictyostelium discoideum
+              value: Dictyostelium discoideum
+        - name: source
+          label: Source
+          type: basic:string
+          custom_choice_conditions: ["geo_annotation.organism"]
+          description: >
+            Briefly identify the biological material e.g., vastus lateralis muscle.
+        - name: cell_type
+          label: Cell type
+          type: basic:string
+          custom_choice_conditions: ["geo_annotation.organism"]
+          hidden: "geo_annotation.organism != 'Homo Sapiens' || geo_annotation.organism != 'Mus musculus'"
+          description: >
+            Type of cell of the sample or from which the sample was obtained.
+        - name: strain
+          label: Strain
+          type: basic:string
+          custom_choice_conditions: ["geo_annotation.organism"]
+          hidden: "geo_annotation.organism != 'Dictyostelium discoideum'"
+          description: >
+            Microbial or eukaryotic strain name.
+        - name: genotype
+          label: Genotype
+          type: basic:string
+          allow_custom_choice: true
+          custom_choice_conditions: ["geo_annotation.organism"]
+          hidden: "geo_annotation.organism != 'Dictyostelium discoideum'"
+            choices:
```

Comment:
YAML validator returns en error here `expected <block end>`. Should `choices` be indent one level to the left?